src/test/scala-2/chiselTests/BoringUtilsSpec.scala

```diff
@@ -689,4 +689,21 @@ class BoringUtilsSpec extends AnyFlatSpec with Matchers with LogUtils with FileC
            |""".stripMargin
       )
   }
+  it should "fail if endIOCreation is set" in {
+
+    class Bar extends RawModule {
+      val baz = Module(new Baz)
+      BoringUtils.bore(baz.c)
+    }
+
+    class Baz extends RawModule {
+      val c = Wire(Property[Int]())
+      endIOCreation()
+    }
+
+    val e = intercept[Exception] {
+      circt.stage.ChiselStage.emitCHIRRTL(new Bar, args)
+    }
+    e.getMessage should include("Cannot create secret ports into Baz (from Bar) if IO creation is not allowed")
```

Comment:
this is a user level API that's doing the wrong thing, but users presumably don't know about "secret ports". Can we make this error message reference the bore? Can we say "bore or tap"?

Reply:
presumably this test needs to be updated

Also is there a reason not to update the older message too? It would be a change, but the same argument applies. Is there any other user-level way of creating "secret ports"?

Reply:
I believe the only way to create "secret ports" is through BoringUtils.